src/main/java/frc/robot/subsystems/Lights.java

```diff
@@ -112,31 +54,66 @@ public void setSpacing(int s) {
   }
 
   /** sets the color of the LEDs in RGB */
-  public void setColor(LightCodes light) {
+  public void setColor(LightCode light) {
     int r = 0;
     int g = 0;
     int b = 0;
-
-    if (light == LightCodes.CONE || light == LightCodes.CUBE || light == LightCodes.WORKING) {
-      r = 255;
-    }
-    if (light == LightCodes.CONE || light == LightCodes.GAME_OBJECT || light == LightCodes.NO_TAG) {
-      g = 255;
-    }
-    if (light == LightCodes.CUBE || light == LightCodes.READY_TO_SCORE) {
-      b = 255;
-    }
-    if (light == LightCodes.OFF) {
-      r = 0;
-      g = 0;
-      b = 0;
+    switch (light) {
+      case CONE:
+        r = 255;
+        g = 255;
+        b = 0;
+        break;
+      case CUBE:
+        r = 255;
+        g = 0;
+        b = 255;
+        break;
+      case GAME_OBJECT:
+        r = 0;
+        g = 255;
+        b = 0;
+        break;
+      case NO_TAG:
+        r = 0;
+        g = 255;
+        b = 0;
+        break;
+      case WORKING:
+        r = 255;
+        g = 0;
+        b = 0;
+        break;
+      case READY_TO_SCORE:
+        r = 0;
+        g = 0;
+        b = 255;
+        break;
+      case OFF:
+        r = 0;
+        g = 0;
+        b = 0;
+        break;
+      default:
+        // this should never trigger
+        r = 0;
+        g = 0;
+        b = 0;
     }
 
-    for (int i = 0; i < m_ledBuffer.getLength(); i = i + spacing) {
-      m_ledBuffer.setRGB(i, r, g, b);
+    for (int i = 0; i < ledBuffer.getLength(); i = i + spacing) {
+      ledBuffer.setRGB(i, r, g, b);
     }
   }
 
   @Override
-  public void periodic() {}
+  public void periodic() {
+    if (currentLightStatus == LightCode.NO_TAG) {
+      if (blinkingTimer == 500) {
+        toggleCode(LightCode.NO_TAG);
+        blinkingTimer = 0;
+      }
+    }
+    blinkingTimer += 50;
```

Comment:
Not sure what 50 and 500 are here. Really what you're doing here is switching every 10 cycles. So you might want a variable equal to 10 with some descriptive name (blinking period or something) and use it that way